client/components/header.jsx

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 
 export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.returnHome = this.returnHome.bind(this);
+  }
+
+  returnHome(event) {
+    this.props.updateDecks();
+    location.href = '/#';
+  }
```

Comment:
🤔 This makes me think that we should move the deck fetching into the deck list component so that it always loads a fresh list in `componentDidMount` would that break anything?

Reply:
It looks like I was able to successfully move the deck storage to the decks.jsx component page. I just rewrote my fetch request when I submit my modal which I was originally passing as a prop from the app.jsx page.

![image](https://user-images.githubusercontent.com/109567499/201793245-1019483d-1063-4d84-a9b0-3c823dd6840b.png)


![deck fetch move](https://user-images.githubusercontent.com/109567499/201793233-88e98b69-deea-40b6-b655-59fbb60d3479.gif)